pinot-core/src/main/java/org/apache/pinot/core/segment/index/metadata/ColumnMetadata.java

```diff
@@ -97,6 +98,7 @@ public static ColumnMetadata fromPropertiesConfiguration(String column, Properti
     builder.setContainsNulls(config.getBoolean(getKeyFor(column, HAS_NULL_VALUE)));
     builder.setHasDictionary(config.getBoolean(getKeyFor(column, HAS_DICTIONARY), true));
     builder.setHasInvertedIndex(config.getBoolean(getKeyFor(column, HAS_INVERTED_INDEX)));
+    builder.setHasFSTIndex(config.getBoolean(getKeyFor(column, HAS_INVERTED_INDEX), false));
```

Comment:
`HAS_FST_INDEX`

Reply:
ah yup that's correct, let me fix it

Reply:
Here is the fix: https://github.com/apache/incubator-pinot/pull/6441